RELEASE/data/autoscend_properties.txt

```diff
@@ -226,3 +226,4 @@
 223	auto_paranoia_counter
 224	auto_wishes
 225	auto_restoreUseBloodBond
+226	auto_jarlsbergSkills
```

Comment:
you should delete this. first line in this file explains why
## This file is used by autoscend_migration.ash to convert cc_ or sl_ type properties into auto_ properties. The old cc_ and sl_ properties are deleted in the process
## If a property is being deleted from autoscend then add its removal to auto_depreciation but leave it here.